lib/core/utils/responsive_helper.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:flutter/material.dart';
+
+class ResponsiveHelper {
+  static const double tabletMinWidth = 800;
```

Comment:
Where did you get this code? From Android, I remember this value was 720.

Reply:
It can change.